dspace/modules/journal-submit/src/main/java/org/datadryad/submission/DryadGmailService.java

```diff
@@ -46,7 +46,7 @@
      * @throws IOException
      */
     public DryadGmailService(File credentialPath, String clientsecretsFile, String scope, String user) {
-        myUserID = user;
```

Comment:
I've tried multiple times on dev -- I'm unable to get it to authenticate with the new change in place, but it works fine with this line as-is.